### PR TITLE
Add Bintray settings to build.gradle

### PR DESCRIPTION
**Pull request by sakama**
There're some SaaS/PaaS cloud services that are using Elasticsearch.
I want to write plugins for these services based on this plugin(want to use this plugin as a library).

### CODE REVIEW

**Comment by muga** [comment]
@sakama thank you for improving. can you consider to transfer this repo under embulk account??

**Comment by sakama** [comment]
@muga Sounds good.
I don't have permission to transfer this repository since I don't have an admin permission.
Could you give me a permission?

**Comment by muga** [comment]
@sakama I transferred this repo under embulk account. please confirm it.

File: build.gradle [hunk 1]
```diff
@@ -35,13 +37,71 @@ dependencies {
...
+
+    publications = ['bintrayMavenRelease']
+    publish = true
+
+    pkg {
+        userOrg = 'embulk-output-elasticsearch'
+        repo = 'maven'
+        name = project.name
+        desc = 'Elasticsearch output plugin for Embulk'
+        websiteUrl = 'https://github.com/muga/embulk-output-elasticsearch'
```
Comment (muga) [inline build.gradle]: https://github.com/embulk/embulk-output-elasticsearch

File: build.gradle [hunk 2]
```diff
@@ -35,13 +37,71 @@ dependencies {
...
+        issueTrackerUrl = 'https://github.com/muga/embulk-output-elasticsearch/issues'
```
Comment (muga) [inline build.gradle]: muga -> embulk

File: build.gradle [hunk 3]
```diff
@@ -35,13 +37,71 @@ dependencies {
...
+        vcsUrl = 'https://github.com/muga/embulk-output-elasticsearch.git'
```
Comment (muga) [inline build.gradle]: muga -> embulk

**Review by muga — APPROVED** [review]
Basically LGTM. Please rename repo in bintray settings. and please remove "[wip]" from the title.

**Comment by sakama** [comment]
Thanks.

I updated repository to "embulk" at build.gradle and created new organization at bintray.
https://bintray.com/embulk-output-elasticsearch

`./bradlew bintrayUpload` command works successfully.
https://bintray.com/embulk-output-elasticsearch/maven/embulk-output-elasticsearch

**Pull request merged**